java/src/hu/holyoil/controller/SunController.java

```diff
@@ -38,18 +39,34 @@ public String toString() {
      * Pseudo random szám, ami a következő napviharig hátralévő köröket számolja
      * <p>körönként csökken,napvihar után újra sorsolódik</p>
      */
-    private Integer turnsUntilNextSunstorm;
+    private int turnsUntilNextSunstorm;
     /**
      * A pályán található összes aszteroidát tartalmazó lista
      */
     private List<Asteroid> asteroids;
 
+    /**
+     * Újraindítja a visszaszámláló turnsUntilNextSunstorm számlálót egy random számra 20 és 50 között
+     */
+    private void RestartCountdown() {
```

Comment:
This should account for randomness enabled

Reply:
Done